src/commands/connections/pool/clear.ts

```diff
@@ -0,0 +1,18 @@
+import { runCommand } from '../../../util/command';
+import { outputFormat } from '../util';
+const open = require('open')
+
+export = {
+	command: 'clear <id>',
```

Comment:
i propose changing this to either delete or remove vs. clear. delete is more consistent throughout the application, but may not exactly reflect the action being taken...

Reply:
Remove sounds better to me. Delete gives a stronger implication that it's deleting the actual connection pool, i.e.: "skysync-cli connections pool delete 2345432632"